Automattic/woocommerce-payments
# Add onboarding payment gateway setup methods

Part of - https://github.com/woocommerce/woocommerce-admin/issues/6878

Adds the methods needed to use the config-based setup for the payment gateway during onboarding.

![image](https://user-images.githubusercontent.com/444632/120547290-bfcf8300-c3a5-11eb-8aab-186ee946c37f.png)


#### Testing instructions
1. Install and activate this plugin in your `plugins/` directory.
2. Install and activate the `add/remote-payment-wcpay` branch of [WooCommerce Admin](https://github.com/woocommerce/woocommerce-admin) ( `git clone https://github.com/woocommerce/woocommerce-admin.git && cd woocommerce-admin  && git checkout add/remote-payment-wcpay && composer install && npm i && npm run build` ).
3. Make sure a qualifying WCPay is selected, and that the "cbd" industry is not selected during onboarding.
4. Navigate to the payment tasks via _WooCommerce -> Home -> Set up payments_
5. Click "Finish Setup" on WCPay card.
6. Make sure that the description text is displayed at the connect button directs you to the correct place.

-------------------


**Post merge**
